pkg/osquery/table/platform_tables_windows.go

```diff
@@ -16,6 +16,8 @@ import (
 
 func platformTables(client *osquery.ExtensionManagerClient, logger log.Logger, currentOsquerydBinaryPath string) []osquery.OsqueryPlugin {
 	return []osquery.OsqueryPlugin{
+		dataflattentable.TablePluginExec(client, logger,
+			"kolide_bitlocker_volumes", dataflattentable.JsonType, []string{"Get-BitlockerVolume", "|", "ConvertTo-Json"}),
```

Comment:
Did you this it? This is not a command -- it's closer to a shell pipeline. Generally speaking, those are not supported. (And I don't know that we support powershell commandlets. Maybe we could investigate that)

Do you know if this data is accessible via other, non-exec, paths?